polaris-react/src/components/Badge/Badge.scss

```diff
@@ -32,19 +25,14 @@
 
 .statusSuccess {
   background-color: var(--p-color-bg-success);
+  color: var(--p-color-text-success);
 
-  #{$se23} & {
-    color: var(--p-color-text-success);
-    background-color: var(--p-color-bg-success);
-
-    // stylelint-disable-next-line -- se23 svg fill override
-    svg {
-      fill: var(--p-color-icon-success);
-    }
+  svg {
+    fill: var(--p-color-icon-success);
   }
 }
 
-@mixin statusSuccess-strong {
+.statusSuccess-strong-experimental {
```

Comment:
Should we also remove `-experimental` from the classname?

Reply:
I'm going to leave this as is for now, it is using the prop variation to get the class name and we aren't cleaning up the props just yet